libpod/define/pod_inspect.go

```diff
@@ -51,6 +51,10 @@ type InspectPodData struct {
 	// Containers gives a brief summary of all containers in the pod and
 	// their current status.
 	Containers []InspectPodContainerInfo `json:"Containers,omitempty"`
+	// CPUs contains the total number of executable CPUs in the pod
+	CPUs float64 `json:"cpus,omitempty"`
```

Comment:
```suggestion
	// CPUs contains the total number of executable CPUs in the pod
	CPUs int `json:"cpus,omitempty"`
```
Partial CPU's isn't supported, Correct? NumCPU() signature is int.

Reply:
@jwhonce I could be wrong but @rhatdan told me float64 in the doc he shared w/ me. The default is 0.000, I guess you can specify a portion of a core?

Reply:
Could have been a cut and paste issue.  But take a look at this field in containers?

Reply:
@rhatdan we figured it out for the most part. Decided to read in the CPUs as a float but then immediately use it to calculate the quota and period and just store those as uints/ints in the intermediate structs.

Reply:
SGTM